### DISCUSSION TITLE
Error Processing Application for Remote Student

### DISCUSSION THREAD

**Discussion by KeraLinn** [community/community#62504] [GitHub Education]
### Hi everyone,

I am located in Arkansas and am actively enrolled in an online degree program from Full Sail University, which is located in Winter Park, Florida. My Enrollment Verification Letter specifically states that the degree program is Online, but when trying to process my application, I get this error:

You appear not to be near any campus location for the school you have selected. Your school-provided academic affiliation document must indicate virtual learning if you are remote. Please enable the application to discover your location from the browser pop-up to provide an accurate location. If you are using a VPN, please disable it when you reapply. If you believe that the school you have applied for has incorrect information that affects your current application e.g. incorrect campus location, please select ‘My selected school has incorrect or incomplete information e.g. domains or campus location’ option when submitting a GitHub Education support ticket.

How else am I supposed to have the document indicate that I am a remote student?

**Comment by Awrsha** [comment]
This is a frustrating situation, but there are a few things you can try to resolve it:

Double check that your Enrollment Verification Letter explicitly states that your degree program is fully online. Make sure it uses that exact phrasing.
GitHub may be looking for very specific language to confirm your remote status.

Resubmit your application and allow the browser pop-up to detect your location. Even though you live far from the physical campus, this may be required for GitHub to approve your application.

Open a support ticket with GitHub Education and explain the situation clearly:

Mention that you are enrolled in an accredited online degree program
Provide the name and location of your university
Specify that your Enrollment Verification Letter states the program is online
Say that you are unable to be near any physical campus location due to the online nature of the program
In the support ticket, ask GitHub Education specifically how they require remote student status to be indicated in the enrollment document. See if they have any other requirements beyond "online" being stated.

Be patient but persistent. Unfortunately GitHub has strict eligibility requirements to minimize abuse, so it may take multiple communications and explanations to verify your remote student status.

The key is clearly communicating with GitHub Education support - explain your accredited online program, provide all relevant documentation, and ask them point-blank how they require remote status to be indicated. Hopefully with a manual review, they'll be able to approve your application as a legitimate remote student.
